src/gui/systray.h

```diff
@@ -34,6 +34,13 @@ namespace Ui {
     class Systray;
 }
 
+enum TaskBarPosition {
```

Comment:
Let's have it "enum class" to force scoping and avoid implicit conversion to int

Also would make sense to have it inside the Systray class and use Q_ENUM to make it easier to expose to QML.

Reply:
✔️